src/styles/_grid.scss

```diff
@@ -0,0 +1,370 @@
+.grid {
+  display: grid;
```

Comment:
Well done! I noticed below 744px the content moves to the left so I would suggest to add justify content center to your grid. Other than that everything looks awesome.

```
.grid {
  display: grid;
  justify-content: center;
  grid-template-columns: repeat(2, 130px);
  gap: 20px;
```